scripts/lib/integration.sh

```diff
@@ -24,12 +24,7 @@ function run_kops_conformance() {
     echo "TIMELINE: KOPS tests took $KOPS_TEST_DURATION seconds."
 
     sleep 240 #Workaround to avoid ENI leakage during cluster deletion: https://github.com/aws/amazon-vpc-cni-k8s/issues/1223
-    START=$SECONDS
-    down-kops-cluster
```

Comment:
Is there a reason we no longer tear down the cluster here? Does that happen elsewhere?

Reply:
Earlier we only used to run the upstream e2e tests. For the version automation, we wanted to run cni tests periodically. So instead of finishing the workflow here, it goes and also runs the tests defined in the script

Reply:
Got it, just wanted to make sure that all resources are cleaned up